### PR TITLE
Monthly Release News for 3.7.3

### PR DESCRIPTION
**Pull request by kmckeedocs**
Here is the content for the Monthly Release News that will be published on 5/5. As part of the migration restructure, agent notes are called out separately and added to the end of this doc. This will change next month. Please note any concerns, **especially regarding the Bug Fixes in the Release News.** (These were written to format and I want to be sure the meaning is still correct).

### CODE REVIEW

File: content/release/ReleaseNotes373.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+# Monthly Release News - April 2020
+
+This page shows highlights from our all Contrast hosted, on-premises and agent releases over the past month.
+
+**3.7.3 on-premises release date: ** May 5, 2020
+
+**New and improved:**
+
+- In addition associating vulnerabilities, you can now also **associate both discovered and exercised routes** to build numbers, application versions, branches or repositories **using session metadata**. This means you can also query route information with a public API endpoint. With a single call to a public endpoint you can get detailed information on how much of an application has been exercised and where the critical vulnerabilities are.
+- You now have a choice to receive individual policy violation emails or to consolidate them into a single email. Find this option under **Organization Settings \&gt; Notifications**.
```
Comment (Silvenga) [inline content/release/ReleaseNotes373.md:16]: Not sure if `>` was supposed to be escaped twice here.

File: content/release/ReleaseNotes373.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+**Important notes:**
+
+- To improve security, Contrast has updated our JRE version to Java 11 for both hosted and on-premises customers. This should not affect end users.
+
+**Bug fixes:**
+
+These significant bugs have been fixed in the past month:
+- SUP-1244 (TS-2697,TS-1494) 3.7.2 upgrade caused Contrast server and mysqld to run as the wrong user.
+- SUP-1153 (JAVA-1051) RBAV was incorrectly auto-remediating vulnerabilities.
```
Comment (scorlett) [inline content/release/ReleaseNotes373.md:27]: "auto-remediating" should be "auto-verifying" (RBAV is Route Based Auto-Verification)

File: content/release/ReleaseNotes373.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+- Your AppSec team can more easily assess library security risk and prioritize work with changes to surface CVE severity and make libraries easier to find. Select **Libraries** to see a **filterable list of libraries with visual display of CVE severity** for each one.
+
+**Important notes:**
+
+- To improve security, Contrast has updated our JRE version to Java 11 for both hosted and on-premises customers. This should not affect end users.
+
+**Bug fixes:**
+
+These significant bugs have been fixed in the past month:
+- SUP-1244 (TS-2697,TS-1494) 3.7.2 upgrade caused Contrast server and mysqld to run as the wrong user.
```
Comment (scorlett) [inline content/release/ReleaseNotes373.md:26]: Maybe I'm being overly pernickety, but I feel this should say:
"3.7.2 upgrade caused Contrast server and mysqld to _attempt to_ run as the wrong user"

File: content/release/ReleaseNotes373.md (hunk 1 above)
Comment (kmckeedocs) [reply content/release/ReleaseNotes373.md:16]: I think you are talking about the clickpath, which should just have one >

File: content/release/ReleaseNotes373.md (hunk 3 above)
Comment (kmckeedocs) [reply content/release/ReleaseNotes373.md:26]: This is the kind of detail I'm hoping for, thank you.

File: content/release/ReleaseNotes373.md (hunk 2 above)
Comment (kmckeedocs) [reply content/release/ReleaseNotes373.md:27]: Done.

**Pull request merged**